Lesson2.ipynb

```diff
@@ -648,6 +589,580 @@
     "    print(\"All good!\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Advanced Types\n",
+    "\n",
+    "The advanced types in Python are also called data structures.\n",
+    "\n",
+    "A data structure is a particular way of organizing and storing data in a computer so that it can be accessed and modified efficiently. (Wikipedia)\n",
+    "\n",
+    "Today, we will learn the following data structures: Lists, Tulpes and Dictionaries."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "### Lists\n",
+    "\n",
+    "As the name indicates, a List is an ordered set of data. It's also called Collections, sequences or Arrays. It is similar to our daily checklist, task list, etc.\n",
+    "\n",
+    "Data is comma-separated and you declare a list using square brackets. It accepts every type of data:\n",
+    "\n",
+    "```python\n",
+    "# An empty list\n",
+    "myList = []\n",
+    "\n",
+    "# A list of integers\n",
+    "myList = [1,2,10,220]\n",
+    "\n",
+    "# A list of integers, floats, strings and booleans\n",
+    "myList = [1,\"2\",10.5, False]\n",
+    "\n",
+    "# A list of Lists! (also called nested lists)\n",
```

Comment:
Should there be a small separate section to explain nesting? If we only leave this within the List section, students may think only lists can be nested.